networkx/readwrite/json_graph/tests/test_adjacency.py

```diff
@@ -4,13 +4,19 @@
 
 import networkx as nx
 from networkx.readwrite.json_graph import adjacency_data, adjacency_graph
+from networkx.utils import graphs_equal
 
 
 class TestAdjacency:
     def test_graph(self):
         G = nx.path_graph(4)
         H = adjacency_graph(adjacency_data(G))
-        nx.is_isomorphic(G, H)
+        assert nx.is_isomorphic(G, H)
```

Comment:
Good catch on the missing asserts, and I agree that `graphs_equal` is the right thing to check. I think `is_isomorphic` then becomes redundant so I'd vote to remove all of the `is_isomorphic` calls in this file in favor of `graphs_equal`.